src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java

```diff
@@ -40,7 +39,9 @@
   public static void enterBigQueryProperties(String tableProp) throws InterruptedException, IOException {
     CdfStudioLocators.bigQueryProperties.click();
     CdfBigQueryPropertiesLocators.bigQueryReferenceName.sendKeys(AUTOMATION_TEST);
-    CdfBigQueryPropertiesLocators.projectID.sendKeys(SeleniumHelper.readParameters(PROJECT_ID));
+    SeleniumHelper.replaceElementValue(CdfBigQueryPropertiesLocators.projectID,
+                                       SeleniumHelper.readParameters(PROJECT_ID));
+    CdfBigQueryPropertiesLocators.dataSetProjectID.sendKeys(SeleniumHelper.readParameters(PROJECT_ID));
```

Comment:
dataSetProjectID->datasetProjectID

Please run the build locally before submitting, this issue would have been caught there!